inc/admin/upgrader.php

```diff
@@ -254,8 +254,6 @@ function rocket_first_install() {
  */
```

Comment:
@Tabrisrp Should this file move into `inc/Engine/Admin`???

Reply:
Do we want to mix old and new code in  the new structure? And also classes vs procedural? We didn't discuss those yet

Reply:
Not everything will change necessarily from procedural to OOP. In some cases, we want to have functions available.  But your point is valid.

Let's think about: Is this upgrader a candidate for refactoring into a class? If yes, then we can schedule that refactoring work for later. If no, then we can retain it's structure but relocate it into the Engine or API. 

What do you think?

Reply:
I believe the whole file could get some benefits by being refactored in a class. It's all internal stuff, so it wouldn't be any value for the API part though.

Reply:
Okay cool. Let's look at this as a separate task. Thanks @Tabrisrp